### ISSUE TITLE
@lingui/cli cannot find module 'babel-core'

### ISSUE THREAD

**Issue by lexkrstn** [lingui/js-lingui#162]
Typing `lingui extract` in the terminal I've got the following:
```
Error: Cannot find module 'babel-core'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\coder\AppData\Roaming\nvm\v8.1.4\node_modules\@lingui\cli\api\extractors\babel.js:10:18)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
```
`@lingui/cli 2.0.2`

I suppose it should be `@babel/core` instead of `babel-core` there.

**Comment by lexkrstn** [comment]
The same problem in:
`@lingui/cli/api/compile.js`
I've just hardcoded the sources in node_modules dir and after replacing all the `babel-*` entries to `@babel/*` the `lingui compile` seems to be working now without any errors.

**Comment by tricoder42** [comment]
Good catch! Don't know how this slipped through tests.

Interested in sending a PR?

**Comment by pcraig3** [comment]
Hey, I'm running into exactly this problem, just following [the current CLI tutorial](https://lingui.github.io/js-lingui/tutorials/cli.html). 

I was planning to stay on `lingui-cli: "^1.4.4"` until this gets resolved.

**Comment by lexkrstn** [comment]
@tricoder42 Sorry, I'd love to, but can't pass though initial tests. They fail with a bunch of oddly looking errors, something like that:
```
    Error: expect(received).toEqual(expected)
      Expected value to equal:
        "const languages = {
        en: \"English\",
        cs: \"Czech\"
      };"
      Received:
        "const languages = {
        en: \"English\",
        cs: \"Czech\"
      };"

      Difference:

      - Expected
      + Received

      - const languages = {
      -   en: "English",
      -   cs: "Czech"
      + const languages = {
      +   en: "English",
      +   cs: "Czech"
        };
````
Don't know what the problem is.

**Comment by tricoder42** [comment]
@lexkrstn Very interesting. You're using Windows? I guess the only problem can be whitespace, all other characters looks fine. Anyway, I'm just testing the fix. Should be published soon (today).

The problem is that Jest still uses `babel-core`, so tests don't fail when resolving incorrect package name.

**Comment by tricoder42** [comment]
Fixed in **2.0.3**

**Comment by dhrubesh** [comment]
![image](https://user-images.githubusercontent.com/11453045/46310104-06102b00-c5dc-11e8-9464-f649864c9987.png)
Hey, I'm still getting this error. Any workaround?
